wrappers/winnt.go

```diff
@@ -1477,6 +1477,7 @@ const (
 	SERVICE_WIN32_SHARE_PROCESS = 0x00000020
 	SERVICE_WIN32               = SERVICE_WIN32_OWN_PROCESS | SERVICE_WIN32_SHARE_PROCESS
 	SERVICE_INTERACTIVE_PROCESS = 0x00000100
+	SERVICE_TYPE_ALL            = SERVICE_WIN32 | SERVICE_ADAPTER | SERVICE_DRIVER | SERVICE_INTERACTIVE_PROCESS
```

Comment:
From what I see at https://docs.microsoft.com/en-us/windows/win32/api/winsvc/nf-winsvc-enumservicesstatusexa, it looks like `SERVICE_TYPE_ALL` is an undocumented constant, and I've been trying to keep undocumented API elements out of this library since they're (at least in theory) subject to change or removal in future Windows versions.

Reply:
You right, this constant is not in the documentation but it is WinNT.h.
Without this constant, everyone using the library will have to define it, and that looks bad too.  Maybe it's better to leave it defined?

Reply:
Right, but if it's undocumented, then presumably very few people are using it.  Let's leave it out for now.  It's easier to add things later if we decide that we want them than to remove something after it's been published.

Reply:
Ok, done.